MAGFit/index.html

```diff
@@ -114,26 +148,46 @@ <h2 style="text-align:center">Expected vs measured body frame magnetic field</h2
 <div id="mag_plot_z" style="width:1200px;height:300px"></div>
 
 <table><tr><td style="width:1200px">
-    <h2 style="text-align:center">Yaw change</h2>
+    <h2 style="text-align:center">Yaw change
+        <img id="TT" src="../images/question-circle.svg" style="width: 1em; vertical-align: bottom" 
+        data-tippy-content='Change in yaw resulting from MAGFit. The first plot is change in yaw between new MAGFit calibration and the existing calibration. 
+                            The second plot it change in yaw between new MAGFit calibration and the selected attitude source.'
+        data-tippy-maxWidth='750px'/>
+    </h2>
 </td></tr></table>
 
 <div id="yaw_change_mag" style="width:1200px;height:300px"></div>
 <div id="yaw_change_att" style="width:1200px;height:300px"></div>
 
 <table><tr><td style="width:1200px">
-    <h2 style="text-align:center">Field length</h2>
+    <h2 style="text-align:center">Field length
+        <img id="TT" src="../images/question-circle.svg" style="width: 1em; vertical-align: bottom" 
+        data-tippy-content='Comparison of felid length &radic;(X&#178; + Y&#178; + Z&#178;). Felid length errors may coincide with current draw.
+                            Significant corelation could mean using battery compensation would give better results, if correlation remains when using a compensated fit a hardware fix may been required.'
+        data-tippy-maxWidth='750px'/>
+    </h2>
 </td></tr></table>
 
 <div id="field_length" style="width:1200px;height:300px"></div>
 
 <table><tr><td style="width:1200px">
-    <h2 style="text-align:center">Motor compensation sources</h2>
+    <h2 style="text-align:center">Motor compensation sources
+        <img id="TT" src="../images/question-circle.svg" style="width: 1em; vertical-align: bottom" 
+        data-tippy-content='Battery current for comparison with field length and fit error. 
+                            Significant corelation could mean using battery compensation would give better results, if correlation remains when using a compensated fit a hardware fix may been required.'
```

Comment:
correlation....and what hardware fix???

Reply:
"hardware fix may been required to move the compass further from the interference source"